landing-page/landing-page.css

```diff
@@ -0,0 +1,235 @@
+@import url('../utilities.css');
+
+.nav-section{
+    background-color: var(--clr-white);
+    box-shadow: var(--box-shadow-clr);
+    justify-content: flex-start;
+    flex-shrink: 2;
+    height: 7rem;
+    max-width: 100vw;
+    padding-inline-start: 1rem;
+    position: fixed;
+    top:0;
+    width: 100%;
+    z-index: 5;
+}
+.logo-container{
+    padding-inline: 2.5rem 1.5rem;
+    padding-bottom: 1rem;
+}
+.menu-btn{
+    position: absolute;
+    top: 2.5rem;
+    left: 2rem;
+    z-index: 10;
+}
+.navbar{
+    display: inline-flex;
+    align-items: center;
+}
+
```

Comment:
Everything Looks Awesome !
Try Styling the sidebar scrollbar 
Example code :

.sidebar::-webkit-scrollbar {
  width: 0.5rem;
  height: 0.5rem;
}

.sidebar::-webkit-scrollbar-thumb {
  background-color: black;
  border-radius: 5px;
}

.sidebar::-webkit-scrollbar-track {
  background-color: lightgray;
}

Reply:
Added Custom Scroll bar. Please check